TheWarWithin/Priorities/DeathKnightBlood.simc

```diff
@@ -1,90 +1,82 @@
 actions+=/ancestral_call,if=buff.dancing_rune_weapon.up
 actions+=/fireblood,if=buff.dancing_rune_weapon.up
 actions+=/potion,if=buff.dancing_rune_weapon.up
 actions+=/vampiric_blood,if=tanking&incoming_damage_5s>=vb_damage&!(buff.dancing_rune_weapon.up|buff.icebound_fortitude.up)
 actions+=/variable,name=rp_deficit_threshold,value=15+(10*talent.relish_in_blood.enabled)+(3*talent.runic_attenuation.enabled)+(spell_targets.heart_strike*talent.heartbreaker.enabled*2)
-actions+=/blood_tap,use_off_gcd=1,if=(rune<=2&rune.time_to_3>gcd.max&charges_fractional>=1.8)
-actions+=/blood_tap,use_off_gcd=1,if=(rune<=1&rune.time_to_3>gcd.max)
 actions+=/raise_dead,use_off_gcd=1
+actions+=/blood_tap,if=(rune<=2&rune.time_to_3>gcd&charges_fractional>=1.8)
+actions+=/blood_tap,if=(rune<=1&rune.time_to_3>gcd)
 actions+=/deaths_caress,if=buff.bone_shield.remains<gcd.max*2
-actions+=/death_strike,if=buff.coagulopathy.up&buff.coagulopathy.remains<=gcd.max*2
+actions+=/death_strike,if=buff.coagulopathy.up&buff.coagulopathy.remains<=gcd
 actions+=/any_dnd,if=!buff.death_and_decay.up
 actions+=/blood_boil,if=dot.blood_plague.remains<gcd.max*2
-actions+=/soul_reaper,if=active_enemies=1&(target.time_to_pct_35<5)&target.time_to_die>(dot.soul_reaper.remains+5)&(!hero_tree.sanlayn|pet.dancing_rune_weapon.remains<5)
-# Use Icebound Fortitude if you're taking sufficient damage and you don't have Dancing Rune Weapon or Vampiric Blood up.
-actions+=/icebound_fortitude,if=tanking&incoming_damage_5s>=ibf_damage&!(buff.dancing_rune_weapon.up|buff.vampiric_blood.up)
-# Use Rune Tap if you're taking sufficient damage and you don't have Dancing Rune Weapon, Vampiric Blood, or Icebound Fortitude up.
-actions+=/rune_tap,if=tanking&incoming_damage_5s>=rt_damage&rune>3&!(buff.dancing_rune_weapon.up|buff.vampiric_blood.up|buff.icebound_fortitude.up)
-actions+=/run_action_list,name=san_drw,strict=1,if=hero_tree.sanlayn&buff.dancing_rune_weapon.up
-actions+=/call_action_list,name=san_cds,strict=1,if=hero_tree.sanlayn
-actions+=/run_action_list,name=sanlayn,strict=1,if=hero_tree.sanlayn
-actions+=/call_action_list,name=db_cds,strict=1,if=!hero_tree.sanlayn
-actions+=/run_action_list,name=deathbringer,strict=1,if=!hero_tree.sanlayn
-
-actions.db_cds=reapers_mark
-actions.db_cds+=/dancing_rune_weapon
-actions.db_cds+=/bonestorm,if=buff.bone_shield.stack>=5&(!talent.shattering_bone.enabled|death_and_decay.ticking)
-actions.db_cds+=/tombstone,if=buff.bone_shield.stack>=8&(!talent.shattering_bone.enabled|death_and_decay.ticking)&cooldown.dancing_rune_weapon.remains>=25
-actions.db_cds+=/abomination_limb,if=!buff.dancing_rune_weapon.up
+actions+=/dancing_rune_weapon
+actions+=/call_action_list,name=deathbringer,strict_if=hero_tree.deathbringer,if=hero_tree.deathbringer
+actions+=/call_action_list,name=san_drw,strict_if=hero_tree.sanlayn,if=hero_tree.sanlayn&buff.dancing_rune_weapon.up
+actions+=/call_action_list,name=sanlayn,strict_if=hero_tree.sanlayn,if=hero_tree.sanlayn
```

Comment:
All 3 of these can be `strict=1,if=`*whatever* -- this is setting up for redundant checks.  All of the criteria here will pass consistently or, in the case of DRW, will only go from true -> false and never false -> true.